templates/server-deployment.yaml

```diff
@@ -121,6 +121,17 @@ spec:
           {{- if or $.Values.server.additionalEnv $serviceValues.additionalEnv }}
           {{- toYaml (default $.Values.server.additionalEnv $serviceValues.additionalEnv) | nindent 12 }}
           {{- end }}
+          {{- if or $serviceValues.additionalEnvCM $serviceValues.additionalEnvSecret }}
+          envFrom:
+            {{- if $serviceValues.additionalEnvCM }}
```

Comment:
Any reason not to handle those as lists? (vs. scalar)

Reply:
I thought that having a unique secret per component would be enough. I can work on handling a list of CM/Secrets.

Reply:
I just updated in order to take lists? @XaF

Reply:
Thanks!